st2actions/st2actions/runners/python_action_wrapper.py

```diff
@@ -16,6 +16,7 @@
 import sys
 import json
 import argparse
+import logging as stdlib_logging
```

Comment:
Not sure why you went with stdlib logging instead of st2common logging that's already used everywhere else. 

You just need to import like so "from st2common import log as logging" in DatsstoreService. Also, passing in a logger seems not needed at that point. Am I missing something?